apps/wallet/src/ui/app/components/recovery-passphrase/RecoveryPassphrase.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useState } from 'react';
+
+import { HideShowDisplayBox } from '_components/HideShowDisplayBox';
+import Alert from '_components/alert';
+import { MenuLayout } from '_components/menu/content/MenuLayout';
+import { PasswordInputDialog } from '_components/menu/content/PasswordInputDialog';
+import { useNextMenuUrl } from '_components/menu/hooks';
+import { useAppDispatch } from '_hooks';
+import { loadEntropyFromKeyring } from '_redux/slices/account';
+import { entropyToMnemonic, toEntropy } from '_shared/utils/bip39';
+
+export function RecoveryPassphrase() {
+    const [passwordConfirmed, setPasswordConfirmed] = useState(false);
+    const [mnemonic, setMnemonic] = useState(['']);
```

Comment:
Can you just default this state to `null` or something?

Reply:
Updated. I did not want to type the `useState` but that makes sense to set as null